applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py

```diff
@@ -38,6 +38,9 @@ def setUp(self):
                         "verbosity": 0
                     }""")
                 ProjectParameters["solver_settings"].AddValue("linear_solver_settings", default_lin_solver_settings)
+            if not ProjectParameters["solver_settings"].Has("silent_warnings_block_builder_and_solver"):
```

Comment:
why adding it this way? this should be done by the default checking within the solver

Reply:
I do not remmeber why :P, I will check